client/pages/stop-profile.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import MapList from './components/list-map';
+
+export default class StopProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      stop: {
+        stopLocation: {},
+        vehicleId: 0,
+        beginTime: null,
+        endTime: null
+      },
+      stopId: this.props.stopId
```

Comment:
Avoid storing your props in state. You can use `this.props.stopId` where you need it.